packages/core/src/tools.ts

```diff
@@ -244,4 +250,33 @@ export namespace Tools {
 		}
 		return position;
 	};
+
+	export const translateText = (str, options?) => {
+		if (!options || !options.rtl) {
+			return str;
+		}
+
+		const arabicNumerals = ["۰", "۱", "۲", "۳", "۴", "۵", "۶", "۷", "۸", "۹"];
```

Comment:
I'm assuming the goal is to translate more than arabic at some point ... maybe add a comment that this isn't permanent?